drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -284,6 +266,24 @@ class DataCollectionStage < Types::BaseEnum
     hud_enum HudUtility2024.data_collection_stages
   end
 
+  class RHYServices < Types::BaseEnum
+    description 'R14.2'
+    graphql_name 'RHYServices'
+    hud_enum HudUtility2024.rhy_services_options
+  end
+
+  class SSVFFinancialAssistance < Types::BaseEnum
+    description 'V3.3'
+    graphql_name 'SSVFFinancialAssistance'
+    hud_enum HudUtility2024.ssvf_financial_assistance_options
+  end
+
+  class EarlyExitReason < Types::BaseEnum
+    description 'R17.A'
+    graphql_name 'EarlyExitReason'
```

Comment:
Welp, it looks like we are not using this enum in the place that we should. `hud_field :early_exit_reason, Types::HmisSchema::Enums::Hud::ExpelledReason` should be using this enum. I can make a note to myself to fix after.